src/governance.h

```diff
@@ -29,17 +39,18 @@ class CGovernanceVote;
 class CNode;
 
 static const CAmount GOVERNANCE_FEE_TX = (0.1*COIN);
-static const int64_t GOVERNANCE_FEE_CONFIRMATIONS = 6;
+static const int64_t GOVERNANCE_FEE_CONFIRMATIONS = 1; //todo 12.1 -- easy testing
```

Comment:
things like this (one value for mainnet and the other for testnet) are №1 candidates for `Params().Consensus` imo